loraxDataTest/CladesOfInterest.txt

```diff
@@ -1,3 +1,4 @@
 Blue_Clams = Unionidae_Rectidentinae_Contradens_contradens, Unionidae_Rectidentinae_Solenaia_khwaenoiensis
-Green_Clams = Etheriidae_Etheriinae_Etheria_elliptica, Notina_TheLoci
-Banana_Clams = Itsa_Notthere, Notina_Theloci
\ No newline at end of file
+Green_Clams = Unionidae_Rectidentinae_Contradens_contradens, Unionidae_Rectidentinae_Solenaia_khwaenoiensis, Notina_TheLoci
+Banana_Clams = Itsa_Notthere, Notina_Theloci
```

Comment:
Banana_Clams: should get 'X' as the 

None of the species in the clade definition are found in the TAXABLE
so the size of the 'relevent species' list for that clade will be 0